__tests__/app.test.js

```diff
@@ -260,3 +260,29 @@ describe("DELETE /api/comments/:comment_id,", () => {
       });
   });
 });
+
+describe("GET /api/users", () => {
+  test("200: returns all the users with the desired categories", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then(({ body }) => {
+        body.forEach((users) => {
```

Comment:
body should not be an array, but an object with a key of in this case `users` which holds the array of users